phpstan.neon.dist

```diff
@@ -137,6 +137,10 @@ parameters:
             message: '~^Method Doctrine\\DBAL\\Schema\\PostgreSQLSchemaManager::fetchTableOptionsByTable\(\) should return array<string, array<string, mixed>> but returns array<int\|string, array<string, mixed>>\.$~'
             paths:
                 - src/Schema/PostgreSQLSchemaManager.php
+        # https://github.com/doctrine/dbal/pull/5835
```

Comment:
When I click on this link, I expect an explanation. Instead I land on this pull request. What's the deal?

Reply:
What about using this https://github.com/doctrine/dbal/pull/5835#discussion_r1056213914 ?

Reply:
Ah yes please do link to the comment.